### DISCUSSION TITLE
Gitbook (New Look) Updates

### DISCUSSION THREAD

**Discussion by vietman78** [GitbookIO/community#580] [0. General bugs and questions]
HI all...i have been reaching out to Gitbook regarding the recent frontend updates. I was super happy with the platform until the recent changes, that appear to have been poorly considered and delivered. Had anyone else had a similar experience as the support team at Giftbook seem to be disinterested in my comments?

**Comment by taszap** [comment]
Hi @vietman78 

Thanks for posting. I am very sorry to hear that you don't think the support team is interested in your concerns. 
My colleague who has spoken with you escalated your concerns to their manager and even shared your video with feedback with our founder.

When we release new changes, we aim to deliver improvements which allow us to ship more advanced updates in the future.
We received a bunch of positive feedback on those changes, although of course, there were also those users who were frustrated.

We take our users' feedback very seriously and always share it with our Product team. 
So far, some of the most common feedback we received about the new version is:

- changes to hint design
- writing (editor) experience being different from published content
- page cover image sizing 
- expendables grouped (in-app and published content)

As my colleague mentioned in #554 we will look to iterate on the readability of content, both in the editor and in published versions, and align the 2 experiences in the future. 

Overall, we believe that the changes offer an improvement to published content although, of course, we will continue to consider any constructive feedback we receive.

**Reply by vietman78** [reply]
Thanks for your feedback but unless you roll back to the previous version of Gitbook, i am not interested. The amount of issues is unacceptable. Its poorly delivered and costing me time and money. We are already having to plan to move on from Gitbook, which will further impact delivery and product consistency. Why Gitbook assumes users want or need these changes is beyond me. These changes have clearly not been tested and have delivered a poor product. As a paid user, i can tell you that i would not use the platform now for FREE it is so bad. I have not seen Gitbook provide anything other than platitudes thus far based on non-forum feedback. So your request for constructive feedback appears to be for appearance rather than any intent to fix your product.

**Reply by SamyPesse** [reply]
Hi @vietman78, this is not appearance. We take the feedback and decide over time what the best actions to take on it are. Sometimes, this is aligned with what our users think, but sometimes, it's not. It also takes time to get feedback from different sources, evaluate it, weigh the pros and cons of different changes, and act on them; while this process is going on, our support team is doing the best they can while communicating with users, and what could be perceived as platitudes or appearance is essentially the tip of the iceberg of what is going on.

> Why Gitbook assumes users want or need these changes is beyond me

You are not the only user of GitBook. This release addressed many feedback and frustrations from other users; if you were not concerned by the changes made in this release, this is unfortunate, but I'd recommend staying away from generalizing your perception without having the full data on how GitBook users use our platform.

> As a paid user, i can tell you that i would not use the platform now for FREE it is so bad.

Point taken, we are sorry to see you leave and we hope you'll find what you need in another solution. Let us know if you need help canceling your account or transitioning your content to a new solution.

The only thing I can tell you at this stage is that we'll always do our best to try to improve our product for all of our users, even if it creates friction; your feedback will help us in that way, and we'll use it to iterate; I genuinely thank you for it and wish you the best.
